ride-service/src/main/kotlin/com/dulfinne/taxi/rideservice/dto/response/RideResponse.kt

```diff
@@ -1,14 +1,15 @@
 package com.dulfinne.taxi.rideservice.dto.response
 
+import com.dulfinne.taxi.rideservice.model.RideStatus
 import java.math.BigDecimal
 import java.time.ZonedDateTime
 
 data class RideResponse(
     var id: Long,
 
-    var driverId: Long?,
+    var driverUsername: String?,
 
-    var passengerId: Long,
+    var passengerUsername: String,
```

Comment:
I don't think that it is a good solution

it's better to stick with ids or to use inner json entity like driver: { //fields }